src/sentry/static/sentry/app/views/settings/team/teamMembers.jsx

```diff
@@ -173,27 +177,37 @@ const TeamMembers = createReactClass({
       .filter(m => !existingMembers.has(m.id))
       .map(m => {
         return {
-          value: m.id,
-          label: m.name || m.email,
+          value: `${m.name} ${m.email}`,
```

Comment:
I'm guessing we're doing this for search? The pattern of `value` and `label` is used throughout the codebase to map item value --> item display name so I think we should keep that here. Instead, we should introduce a new key to use for searching.

Reply:
i agree. Jess got super confused by this and the search actually never worked until I fixed it haha